resources/views/help/developer.twig

```diff
@@ -0,0 +1,127 @@
+{% extends "layouts.master" %}
+
+{% block title %}Help for developers{% endblock %}
+
+{% block content %}
+<ol class="breadcrumb">
+    <li>{{ link_to_route('projects.index', 'Home') }}</li>
+    <li class="active">Help for developers</li>
+</ol>
+
+<h2>Help and guidance for developers</h2>
```

Comment:
This can be removed I think, it's right under breadcrumbs which say what page it is.